resources/utilities/upload_vcf.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
+
+"""
+Usage: $0 PATIENT_ID VCF CREDENTIALS_FILE SERVER_NAME
+
+Upload the VCF file as an attachment for the given patient. For example:
+
+    $0 P0000102 path/to/file.vcf path/to/credentials.txt server_name
+
+credentials.txt should have one line with the form:
+
+    username:password
+"""
+
+from __future__ import with_statement
+
+import sys
+import os
+import logging
+import json
+
+try:
+    from httplib import HTTPConnection
```

Comment:
This works in python2, but not in python3. However, if the script runs in python2, the imported class is HTTPConnection, while in python2 HTTPS_S_Connection is imported. The code below only works with HTTPSConnection, so under python2 it will fail with `NameError: global name 'HTTPSConnection' is not defined`.